### PR TITLE
Update to 4.1

### PR DESCRIPTION
**Pull request by mfairchild365**
Example: http://ucommfairchild.unl.edu:8001/

### CODE REVIEW

**Comment by kabel** [comment]
Let's wait a moment so we can remove the repos and rely on packagist.

**Comment by mfairchild365** [comment]
Yep, I don't think we should merge any of these until we are closer to the launch date. This is just me preparing.

File: composer.json [hunk 1]
```diff
@@ -8,11 +8,18 @@
         {
             "type": "vcs",
             "url": "https://github.com/unl/phpunltemplates.git"
+        },
+        {
+            "type": "vcs",
+            "url": "https://github.com/unl/phpdwtparser.git"
```
Comment (kabel) [inline composer.json]: Should be okay to remove both of these repos.

File: composer.json [hunk 2]
```diff
@@ -8,11 +8,18 @@
...
         }
     ],
     "require": {
-        "pear-unl/UNL_DWT": "*",
+        "unl/php-dwt-parser": "1.0.0-rc.2",
```
Comment (kabel) [inline composer.json]: Should be okay to remove the requirement. It will be pulled from other requirements.

File: composer.json [hunk 3]
```diff
@@ -8,11 +8,18 @@
...
         "pear-unl/UNL_Cache_Lite" : "*",
-        "UNL/PHPUNLTemplates": "dev-master"
-    }
+        "unl/php-wdn-templates": "dev-master"
```
Comment (kabel) [inline composer.json]: Version 4.1.0 is shipped on packagist.

File: composer.json [hunk 4]
```diff
@@ -8,11 +8,18 @@
...
+    },
+    
+    "minimum-stability": "RC",
```
Comment (kabel) [inline composer.json]: No need for RCs now.

File: www/themes/unl/html/SiteMaster/Core/localfooter.tpl.php [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+                <li><a href="http://wdn.unl.edu/">Web Developer Network</a></li>
+                <li><a href="http://iim.unl.edu/">Internet and Interactive Media</a></li>
+                <li><a href="http://ucomm.unl.edu/">University Communications</a></li>
+                <li><a href="http://its.unl.edu/">Information Technology Services</a></li>
+            </ul>
+        </div>
+    </div>
+</div>
+
+<script type="text/javascript">
```
Comment (kabel) [inline www/themes/unl/html/SiteMaster/Core/localfooter.tpl.php]: This should be cleaned up to use the universal analytics code.

File: www/themes/unl/html/SiteMaster/Core/Controller.tpl.php [hunk 6]
```diff
@@ -1,22 +1,34 @@
 <?php
-\UNL_Templates::$options['version']        = 4.0;
-\UNL_Templates::$options['sharedcodepath'] = dirname(__FILE__).'/sharedcode';
```
Comment (kabel) [inline www/themes/unl/html/SiteMaster/Core/Controller.tpl.php]: This reminds me about something I should document. The new templates package works out-of-the-box without a copy of the WDN includes. The previous release automatically failed back to GitHub, which was not ideal. The new package has the `setLocalIncludePath` method to specify the path to the directory that contains the synced `wdn` directory. This replaces the static configuration `Templates::$options['templatedependentspath']`.

You _can_ use ` $page->setLocalIncludePath('https://unlcms.unl.edu');` to use UNLcms's synced directory. But that will require a lot of network requests for _every_ request to your site. It's best to set this to a locally managed directory.

**Pull request merged**